cmd/bridge/main.go

```diff
@@ -129,6 +132,11 @@ func main() {
 		baseURL = bridge.ValidateFlagIsURL("base-address", *fBaseAddress)
 	}
 
+	helmChartRepoURL := &url.URL{}
```

Comment:
Possible to move this logic under your `pkg/helm`?  Something along the lines of:

```go
helmChartRepoURL := setHelmURL(args)
```
Ideally `main.go` should not grow, I'd like it to shrink from now on. :)